docs/articles/apex/testing.md

```diff
@@ -2,27 +2,39 @@
 title: Apex Tests
 ---
 
-You can run Apex tests from within a file or from the Apex Tests sidebar. The sidebar provides other useful features for working with your tests.
+Apex extension allows you run Apex tests to verify the functionality of your code. You can retrieve code coverage results for your Apex classes and triggers every time you run one or more tests. To do this, edit your workspace settings and set `salesforcedx-vscode-core.retrieve-test-code-coverage` to **true**.
 
-## Explore Your Apex Tests
+## Explore Apex Tests
 
-The Apex Tests sidebar provides several features. Here, you can see all your Apex tests at a glance. You can run one test method, the test methods in one class, or all your tests. You can view the results of your last test run. And you can jump from those results to the corresponding lines in your code. To access this sidebar, click the beaker icon (hover text: Test) in the view bar on the left side of the VS Code window. (If you don’t see this icon, make sure that the project you have open in VS Code contains an `sfdx-project.json` file in its root directory.)
+Apex Tests view in the Side Bar provides several features for working with your tests such as run one test method, test methods in one class, or all your tests. To access Apex Tests view, click the beaker icon ({% octicon beaker %}) in the Activity Bar on the left side of the VS Code editor. If you don’t see this icon, make sure that the project contains an `sfdx-project.json` file in its root directory.
```

Comment:
I'm wondering if we should be calling the test sidebar the "Apex Tests" view since the test view displays both apex and lwc tests now. But maybe that's overthinking it. @smaddox-sf

Reply:
Should we link the sfdx project json section under the first project doc?

Reply:
That's a good point @brpowell .  I think we should just call it 'Tests' when we refer to the sidebar since it is more than Apex